CanvasColorSpaceProposal.md

```diff
@@ -140,18 +140,23 @@ IDL
 <pre>
 typedef (Uint8ClampedArray or Uint16ClampedArray or Float32Array) ImageDataArray;
 
-[Constructor(unsigned long sw, unsigned long sh, optional CanvasColorSpace colorSpace = "legacy-srgb"),
- Constructor(ImageDataArray data, unsigned long sw, optional unsigned long sh, optional CanvasColorSpace colorSpace),
+dictionary ImageDataColorSettings {
+  CanvasColorSpace colorSpace = "srgb";
+  CanvasPixelFormat pixelFormat = "8-8-8-8";
```

Comment:
> how PutImageData knows if the p3 is in 10-10-10-2 pixel format or 12-12-12-12 pixel format?

The ImageData is in 16-16-16-16 format, which is implied in the Uint16ClampedArray. putImageData will convert pixels to what ever format the canvas is in (e.g. 10-10-10-2 or 12-12-12-12)